rudder-webapp/SOURCES/rudder-node-to-relay

```diff
@@ -25,19 +25,38 @@ set -e
 # documentation that, without modification of the Source Code, enables
 # supplementary functions or services in addition to those offered by
 # the Software.
-# 
+#
 # Rudder is distributed in the hope that it will be useful,
 # but WITHOUT ANY WARRANTY; without even the implied warranty of
 # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 # GNU General Public License for more details.
-# 
+#
 # You should have received a copy of the GNU General Public License
 # along with Rudder.  If not, see <http://www.gnu.org/licenses/>.
 #
 #####################################################################################
 
 ECHO=/bin/echo
 RUDDER_OPT=/opt/rudder
+RUDDER_VAR=/var/rudder
+RUDDER_JSON="${RUDDER_VAR}/cfengine-community/inputs/rudder.json"
+
+rudder_json_value() {
+	cat "${RUDDER_JSON}" | jq -r ".$1"
```

Comment:
"cat file |" is an anti pattern, use "jq .. < file", even "jq .. file" since it is supported
moreover using -r with jq seems uselss to me

Reply:
my bad i mixed it with -R